### PR TITLE
Add block

### PR DESCRIPTION
**Pull request by ahuang11**
## Summary

Turn credentials into block

### CODE REVIEW

File: prefect_slack/credentials.py [hunk 1]
```diff
@@ -1,43 +1,62 @@
...
     Args:
         token: Bot user OAuth token for the Slack app used to perform actions.
-    """
 
-    token: str
+    Example:
+        Load stored Slack credentials:
+        ```python
+        from prefect_slack import SlackCredentials
+        slack_credentials_block = SlackCredentials.load("MY_BLOCK_NAME")
```
Comment (ahuang11) [inline prefect_slack/credentials.py:20]:
```suggestion
        slack_credentials_block = SlackCredentials.load("BLOCK_NAME")
```

**Review by desertaxle — DISMISSED** [review]
The `SlackWebhook` block here will have the same slug as the `SlackWebhook` block in the core library, which could cause issues. I think we may need to coordinate removing `SlackWebhook` from the core library and adding this one here. I'm fine with merging this now, but we'll need to coordinate with the core library before we release these changes.

**Comment by ahuang11** [comment]
Does this slack webhook need a notify method?

File: prefect_slack/credentials.py [hunk 2]
```diff
@@ -1,43 +1,73 @@
...
 
     Args:
         url: Slack webhook URL which can be used to send messages
             (e.g. `https://hooks.slack.com/XXX`).
+
+    Example:
+        Load stored Slack webhook:
+        ```python
+        from prefect_slack import SlackWebhook
+        slack_webhook_block = SlackWebhook.load("MY_BLOCK_NAME")
```
Comment (desertaxle) [inline prefect_slack/credentials.py:51]:
```suggestion
        slack_webhook_block = SlackWebhook.load("BLOCK_NAME")
```

**Review by desertaxle — APPROVED** [review]
Looks good after one last change.

**Review by desertaxle — APPROVED** [review]

**Pull request merged**